org.civicrm.reqangular/src/common/templates/tab-outer.html

```diff
@@ -0,0 +1,14 @@
+<div>
+  <!--Overrides the 'uib/template/tabs/tabset.html'
+to add '{{tabset.customHeaderClass}}' which is used for custom header class-->
+  <ul class="nav nav-{{tabset.type || 'tabs'}} {{tabset.customHeaderClass}}"
+      ng-class="{'nav-stacked': vertical, 'nav-justified': justified}"
+      ng-transclude></ul>
+  <div class="tab-content">
+    <div class="tab-pane"
+         ng-repeat="tab in tabset.tabs"
+         ng-class="{active: tabset.active === tab.index}"
+         uib-tab-content-transclude="tab">
+    </div>
+  </div>
+</div>
```

Comment:
The advantage of modifying the existing template over completely rewriting it, is that if we want to update the library, we have to keep syncing the original template with our overriding version. Using the original template would make things more maintainable